azurerm/internal/services/compute/virtual_machine_scale_set_resource.go

```diff
@@ -778,6 +778,13 @@ func resourceArmVirtualMachineScaleSet() *schema.Resource {
 			},
 
 			"tags": tags.Schema(),
+
+			"encryption_at_host": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: false,
+				Default:  false,
+			},
```

Comment:
The old `azurerm_virtual_machine_scale_set` resource is currently in the feature-frozen state, it should not receive any new features. You can find the information [here](https://www.terraform.io/docs/providers/azurerm/r/virtual_machine_scale_set.html).

Therefore could we revert the feature changes in this file (such like this)?